samples/matter/lock/prj.conf

```diff
@@ -42,3 +42,6 @@ CONFIG_CHIP_APP_LOG_LEVEL=3
 # Enable Factory Data feature
 CONFIG_CHIP_FACTORY_DATA=y
 CONFIG_CHIP_FACTORY_DATA_BUILD=y
+
+# Enable read client for needed for TimeSyncClient feature
+CONFIG_CHIP_ENABLE_READ_CLIENT=y
```

Comment:
Is it for debugging purposes? I guess we shouldn't enable by default optional features.

Reply:
The TimeSync cluster is configured as a client, so I guess it cannot work without a read client functionality.

Reply:
Well, after a second thought, do we really want to be a client and server? Wouldn't server role be enough for us?

Reply:
To support all required Time Sync features I believe we need to be both server and client. The latter is needed for Time Sync Client feature (the device proactively queries trusted nodes about the current time after joining the fabric).